### PR TITLE
Form styling and cleanup

### PR DESCRIPTION
**Pull request by georgemblack**
Forms across the website have been updated.

screencapture-localhost-3000-users-invitation-accept-5-1512156483643

Forms/elements updated:
* New announcement form
* New event form
* Invite new user form
* Document upload form
* Forgotten password form
* Settings page
* Login
* Reset password form
* Account creation form
* Search bars

Out of scope for this PR:

* Modal rendering
* Styling for specific pages (like permissions, homepage, etc)
* Form contents

### CODE REVIEW

**Comment by Chaseshak** [comment]
@georgemblack Do you want to put some other UI tune-ups (such as the color scheme changes) into this branch as well? If you need help with any of them, let me know. The only other thing I'm working on atm is tests.

**Comment by georgemblack** [comment]
Will do!

File: app/views/announcement/_new.html.erb [hunk 1]
```diff
@@ -1,16 +1,21 @@
 <%= form_for @announcement, url: {action: "create"} do |f| %>
```
Comment (rbrezynski) [inline app/views/announcement/_new.html.erb]: This form goes off the page. Cannot scroll to reach bottom of form, only way to submit is to zoom out of page
screen shot 2017-12-01 at 2 27 41 pm

File: app/views/devise/invitations/edit.html.erb [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-
-  <p><%= f.submit t("devise.invitations.edit.submit_button"), class: 'button-primary' %></p>
-<% end %>
+<div class="info-block-wrapper centered">
+    <div class="info-block">
+        <h2><%= t 'devise.invitations.edit.header' %></h2>
+        <%= form_for resource, :as => resource_name, :url => invitation_path(resource_name), :html => { :method => :put } do |f| %>
+            <%= devise_error_messages! %>
+            <%= f.hidden_field :invitation_token %>
+            <%= f.label 'First Name' %>
```
Comment (rbrezynski) [inline app/views/devise/invitations/edit.html.erb]: Have this text field in focus when first enter page so user knows where to start. All other forms have this so we should be consistent

**Comment by Chaseshak** [comment]
On the permissions page the search bar doesn't have the new form styling. This is because it's not part of a `form` element. Any idea how to apply the CSS to this bar as well?

File: app/views/devise/invitations/edit.html.erb (hunk 2 above)
Comment (georgemblack) [reply app/views/devise/invitations/edit.html.erb]: I'lI look into this and figure it out!

File: app/views/announcement/_new.html.erb (hunk 1 above)
Comment (georgemblack) [reply app/views/announcement/_new.html.erb]: For a separate PR – the modals need to update for different screen sizes

**Comment by georgemblack** [comment]
@Chaseshak thanks for reminding me of this! I'll include search bar styling in this PR as well

**Comment by rbrezynski** [comment]
Need spacing between sorting / invite new user buttons and search bar
"Sort by" font is different than rest of site. 
screen shot 2017-12-02 at 1 16 41 pm

**Pull request merged**